modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/headless/batch/engine/resource/v1_0/test/ExportTaskResourceTest.java

```diff
@@ -145,7 +251,7 @@ private void _testPostExportTask(
 			}
 		}
 
-		Assert.assertEquals(expectedExecuteStatus, actualExecuteStatus);
+		return jsonObject;
 	}
```

Comment:
I think it was good to have the expected status too. For your test, I guess we expect always a SUCCESS, so why don't we also call with the SUCCESS status?